docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -72,6 +72,27 @@ public void publishMessage() {
 }
 ----
 
+==== Subscribing to a subscription
```

Comment:
Maybe, "Asynchronously reading messages from a subscription"?

Reply:
I would like to keep the title as-is because I think it's more idiomatic, but I added some text to the paragraph mentioning subscribing is asynchronous.